tools/kf_mskcc_vcf2maf.cwl

```diff
@@ -58,7 +58,7 @@ inputs:
   ref_build: {type: string?, doc: "Genome ref build used, should line up with cache.", default: "GRCh38"}
   retain_info: {type: string?, doc: "csv string with INFO fields that you want to keep, i.e. for consensus `MQ,MQ0,CAL,Hotspot`"}
   retain_fmt: {type: string?, doc: "csv string with FORMAT fields that you want to keep"}
-  use_kf_fields: {type: boolean?, doc: "Flag to drop fields normally not used in KF, or keep cBio defaults", default: true}
+  maf_center: {type: string?, doc: "Sequencing center of variant called", default: "."}
```

Comment:
Did I not look at another PR recently that had all this `use_kf_fields` -> `maf_center` stuff in it? I don't think this is related to the INFO/AF thing, I just want to draw attention to in in case it is not wanted for this PR.

Reply:
It is! I agree this should have been part of the last PR anyway, perhaps I am a commit behind. Hopefully this should not cause too much of an issue. The tool seemed updated, so not sure why it is not here...

Reply:
I'll pull the main branch, merge, and push to be safe